src/Saas.Permissions/Saas.Permissions.Api/Program.cs

```diff
@@ -1,9 +1,13 @@
+using Microsoft.AspNetCore.Authentication.Certificate;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.Identity.Web;
 using Saas.Permissions.Api.Data;
 using Saas.Permissions.Api.Interfaces;
 using Saas.Permissions.Api.Services;
 
 var builder = WebApplication.CreateBuilder(args);
 
+builder.Services.AddSingleton<IConfiguration>(builder.Configuration);
```

Comment:
This is a sign of container abuse coming right up.

Reply:
same, use options pattern, lets not setup config dependencies everywhere